src/parse_data.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+from typing import Generator, Union, List, Dict, Any
+from datetime import datetime
+
+logger = logging.getLogger('parseStoriesInfo')
+
+def build_stories_info(stories_generator: Generator[dict, None, None]) -> str:
+    logger.info('Parse stories data')
+    stories_info: Dict[str, Any] = dict()
+
+    now = datetime.now()    
```

Comment:
Each time you run a DAG this date is different. How are you going to distinguish the same dag run retry and the other dag run? Usually, airflow's 'exec_date' is used for idempotency.

Reply:
Fix it